### PR TITLE
Postpartum depression cause model

### PR DESCRIPTION
**Pull request by alibow**

### CODE REVIEW

File: docs/source/gbd2019_models/causes/postpartum_depression/index.rst [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+GBD 2019 Modeling Strategy
+--------------------------
+
+Postpartum depression is not specifically estimated by the GBD. Rather, we will inform the prevalence of postpartum depression from a recent review and meta-analysis performed by [Shorey-et-al-2018]_. This review focused on healthy mothers with no prior history of postpartum depression, but found similar results to previous evaluations of postpartum depression prevalence among the broader population. [Shorey-et-al-2018]_ reported that the incidence of postpartum depression was equal to 12% (95% CI 0.04–0.20).
+
+However, the GBD estimated that the average duration of major depressive disorder was 0.65 (95% UI: 0.59, 0.70) of one year (as informed through analysis of longitudinal studies on remission from major depressive disorder and the assumption that 40 years is the maximum duration of the condition), which we will use to inform the duration of postpartum depression in our simulation [GBD-2019-Capstone-Appendix-PPD]_ (page 1013).
```
Comment (zmbc) [inline docs/source/gbd2019_models/causes/postpartum_depression/index.rst:37]: Why does this start with "however"?

File: docs/source/gbd2019_models/causes/postpartum_depression/index.rst [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+However, the GBD estimated that the average duration of major depressive disorder was 0.65 (95% UI: 0.59, 0.70) of one year (as informed through analysis of longitudinal studies on remission from major depressive disorder and the assumption that 40 years is the maximum duration of the condition), which we will use to inform the duration of postpartum depression in our simulation [GBD-2019-Capstone-Appendix-PPD]_ (page 1013). 
+
+Additionally, the GBD models severity distributions and associated disability weights for major depressive disorder, summarized in the table below [GBD-2019-Capstone-Appendix-PPD]_ (page 1013). Notably, the GBD defines major depressive disorder (MDD) as an episodic mood disorder involving the experience of one or more major depressive episode(s). Notably, the percent of case severity was determined from the US National Epidemiological Survey on Alchol and Related Conditions (conducted in two waves from 2001-2002 and 2004-2005) and the Australian National Survey of Mental Health and Wellbeing of Adults. Notably, burden due to major depressive disorder in GBD begins in the 1 to 4 year age group.
```
Comment (zmbc) [inline docs/source/gbd2019_models/causes/postpartum_depression/index.rst:39]:
```suggestion
Additionally, the GBD models severity distributions and associated disability weights for major depressive disorder, summarized in the table below [GBD-2019-Capstone-Appendix-PPD]_ (page 1013). Notably, the GBD defines major depressive disorder (MDD) as an episodic mood disorder involving the experience of one or more major depressive episode(s). The percent of case severity was determined from the US National Epidemiological Survey on Alchol and Related Conditions (conducted in two waves from 2001-2002 and 2004-2005) and the Australian National Survey of Mental Health and Wellbeing of Adults. Notably, burden due to major depressive disorder in GBD begins in the 1 to 4 year age group.
```

I think this is here to imply a limitation that I suggest we make more explicit (see below).

File: docs/source/gbd2019_models/causes/postpartum_depression/index.rst [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+- We are limited in that the rate of PPD in our model is informed from a systematic literature review and meta-analysis that is not location-, year-, or age-specific and the analysis of PPD incidence primarily conducted in high-resource settings.
+
+- We are limited in that we do not consider mortality associated with PPD in our model.
+
+- We are limited in that we assume all PPD cases persist for the same average duration of a single MDD episode. This is limited in the sense that duration may vary by MDD severity (for example, a longer duration for more severe cases), which could cause our estimation of YLDs to be biased. Additionally, we assume that duration of PPD is equal to the duration of all MDD episodes.
+
+- We assume that the onset of PPD occurs immediately following birth. However, the onset of PPD may peak around two or three months postpartum [Shorey-et-al-2018]_.
```
Comment (zmbc) [inline docs/source/gbd2019_models/causes/postpartum_depression/index.rst:79]: Additional limitation: We assume that the GBD MDD severity distribution, which is based on analysis of high-resource settings, generalizes to the severity of PPD in our simulation population of interest.

File: docs/source/gbd2019_models/causes/postpartum_depression/index.rst [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+     - 10 to 14 (ID=7)
+     -
+   * - YLD age group end
+     - 55 to 59 (ID=16)
+     - Note that this is older than able to *become* pregnant, but PPD may extend into this age group among pregnancies that begin in the 50 to 54 year age group. 
+
+Frequency
+"""""""""
+
+Ratios of maternal hemorrhage mortality and incidence are defined in the table below. These values should represent the probability that a simulant experiences a postpartum depression episode **at the time of birth** in our simulation (see the :ref:`pregnancy model <other_models_pregnancy>` for details).
```
Comment (zmbc) [inline docs/source/gbd2019_models/causes/postpartum_depression/index.rst:133]:
```suggestion
This incidence value should represent the probability that a simulant experiences a postpartum depression episode **at the time of birth** in our simulation (see the :ref:`pregnancy model <other_models_pregnancy>` for details).
```

File: docs/source/gbd2019_models/causes/postpartum_depression/index.rst [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+  * - Severe
+    - 10 
+    - 0.658 (0.477, 0.807)
+
+Validation Criteria
+++++++++++++++++++++
+
+- Our simulation should replicate input data for PPD incidence and duration.
+
+- We do not have a validation target for postpartum depression YLDs specifically. However, prevalence should be compared to to values reportd in [Shorey-et-al-2018]_. Additionally, we should compare PPD YLDs per PPD case in our simulation to the implied value from GBD for the major depressive disoders cause.
```
Comment (zmbc) [inline docs/source/gbd2019_models/causes/postpartum_depression/index.rst:179]:
```suggestion
- We do not have a validation target for postpartum depression YLDs specifically. However, prevalence should be compared to to values reported in [Shorey-et-al-2018]_. Additionally, we should compare PPD YLDs per PPD case in our simulation to the implied value from GBD for the major depressive disorders cause. 
```

**Review by zmbc — APPROVED** [review]

File: docs/source/gbd2019_models/causes/postpartum_depression/index.rst [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+
+Additionally, the GBD models severity distributions and associated disability weights for major depressive disorder, summarized in the table below [GBD-2019-Capstone-Appendix-PPD]_ (page 1013). Notably, the GBD defines major depressive disorder (MDD) as an episodic mood disorder involving the experience of one or more major depressive episode(s). Notably, the percent of case severity was determined from the US National Epidemiological Survey on Alchol and Related Conditions (conducted in two waves from 2001-2002 and 2004-2005) and the Australian National Survey of Mental Health and Wellbeing of Adults. Notably, burden due to major depressive disorder in GBD begins in the 1 to 4 year age group.
+
+.. list-table:: Major depressive disorder disability weights
+  :header-rows: 1
+
+  * - Severity
+    - Percent of cases
+    - Disability weight
+  * - Asymptomatic
```
Comment (aflaxman) [inline docs/source/gbd2019_models/causes/postpartum_depression/index.rst:47]: Is GBD's construct of Asymptomatic MDD relevant to PPD?  Maybe we should redistribute this 13% to states with non-zero disability weight.

**Review by aflaxman — APPROVED** [review]

File: docs/source/gbd2019_models/causes/postpartum_depression/index.rst (hunk 4 above)
Comment (alibow) [reply docs/source/gbd2019_models/causes/postpartum_depression/index.rst:133]: Thanks for catching this!

File: docs/source/gbd2019_models/causes/postpartum_depression/index.rst (hunk 6 above)
Comment (alibow) [reply docs/source/gbd2019_models/causes/postpartum_depression/index.rst:47]: Hmm, PPD shares the same diagnostic criteria as MDD in the DSM, so I haven't come across information that would suggest a different diagnostic scheme. I think that "asymptomatic" in GBD might refer to asymptomatic in terms of YLDs (DW=0), but not asymptomatic in that they are not displaying symptoms of MDD. 

The description of mild MDD is as follows: "Feels persistent sadness and has lost interest in usual activities. The person sometimes sleeps badly, feels tired, or has trouble concentrating but still manages to function in daily life with extra effort." I imagine that the asymptomatic sequelae could be something similar, but specifies "with little to no extra effort" instead. I know the diagnostic instruments first assess presence/frequency of symptoms and then assess severity in terms of the degree to which it interferes with daily functioning, so I think the latter portion could be part of how they distinguish asymptomatic and mild cases.

This is not clear in the methods appendix, though, so I'll see if I can get some more info from the GBD modeler to make sure!

File: docs/source/gbd2019_models/causes/postpartum_depression/index.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (alibow) [reply docs/source/gbd2019_models/causes/postpartum_depression/index.rst:79]: Good question! I should add some information for the engineers about whether we should use a fixed propensity across pregnancies or not. Prior depression is a risk factor for PPD, but I think for simplicity we'll use a new propensity for each pregnancy and add this to the limitations. We will introduce some slight recurrence through the hemoglobin risk factor though.

Comment (alibow) [reply docs/source/gbd2019_models/causes/postpartum_depression/index.rst:79]: Yes, this is good to call out! Thanks, I will add it.

File: docs/source/gbd2019_models/causes/postpartum_depression/index.rst (hunk 1 above)
Comment (alibow) [reply docs/source/gbd2019_models/causes/postpartum_depression/index.rst]: I rephrased, thanks :)

**Pull request merged**